src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/IndexWriter.java

```diff
@@ -350,7 +350,7 @@ protected void addLinksForIndexes(List<Character> allFirstCharacters, Content co
         }
 
         content.add(new HtmlTree(TagName.BR));
-        List<Content> pageLinks = Stream.of(IndexItem.Category.values())
+        List<? extends Content> pageLinks = Stream.of(IndexItem.Category.values())
```

Comment:
`List<? extends Content>` is not wrong and is better than `List<Content>` but would `var` be simpler and even better?